### PR TITLE
Fix inconsistency in parameter naming for tf.nn.conv1d

### LINKED ISSUE #6379: Inconsistency in parameter naming
```
tf.nn.conv1d(value, filters, stride, padding, use_cudnn_on_gpu=None, data_format=None, name=None)
tf.nn.conv2d(input, filter, strides, padding, use_cudnn_on_gpu=None, data_format=None, name=None)
tf.nn.conv3d(input, filter, strides, padding, name=None)
```

There's discrepancy between parameter naming between conv1d and 2,3d counterparts. Firstly value vs input, and then singular vs plural filter(s) (singular would be correct as in 2d/3s case since function accepts only single filter if I understood documentation correctly.)

### PR DESCRIPTION
**Pull request by yongtang**
This fix tries to address the issue raised in #6379 where parameter namings for tf.nn.conv1d/conv2d/conv3d are inconsistent. While tf.nn.conv2d/conv3d (and tf.nn.convolution) uses `input` and `filter`, conv1d uses `value` and `filters`.

This fix addresses the issue by deprecating `value` and `filters`, using `input` and `filter`, and, at the same time, maintains backward-compatibility so that current user does not need to change any existing code base.

This fix fixes #6379.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by josh11b** [comment]
Thanks for the PR! API owners wonders if the benefit outweighs the churn here. We'd like to open this up to the wider community to see if there is support for this change (looks like there is a single thumbs up on the original issue at the moment).

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jhseu: It has been 14 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jhseu: It has been 29 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jhseu: It has been 46 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.
